executor/common.h

```diff
@@ -467,8 +469,13 @@ static uintptr_t syz_extract_tcp_res(uintptr_t a0, uintptr_t a1, uintptr_t a2)
 	}
 
 	struct tcp_resources* res = (struct tcp_resources*)a0;
+#if defined(SYZ_EXECUTOR) || defined(SYZ_HANDLE_SEGV)
```

Comment:
One NONFAILING per line will introduce some clutter. Instead of this:
```
	struct tss16* tss16 = (struct tss16*)(host_mem + seg_tss16_2.base);
	NONFAILING(
	    struct tss16* tss = tss16;
	    memset(tss, 0, sizeof(*tss));
	    tss->ss0 = tss->ss1 = tss->ss2 = SEL_DS16;
	    tss->sp0 = tss->sp1 = tss->sp2 = ADDR_STACK0;
	    tss->ip = ADDR_VAR_USER_CODE2;
	    tss->flags = (1 << 1);
	    tss->cs = SEL_CS16;
	    tss->es = tss->ds = tss->ss = SEL_DS16;
	    tss->ldt = SEL_LDT);
```
I would do:
```
struct tss16 tss;
memset(tss, 0, sizeof(tss));
tss.ss0 = tss.ss1 = tss.ss2 = SEL_DS16;
tss.sp0 = tss.sp1 = tss.sp2 = ADDR_STACK0;
tss.ip = ADDR_VAR_USER_CODE2;
tss.flags = (1 << 1);
tss.cs = SEL_CS16;
tss.es = tss.ds = tss.ss = SEL_DS16;
tss.ldt = SEL_LDT;
NONFAILING(host_mem + seg_tss16_2.base, &tss, sizeof(tss));
```
Looks clean and simple.